manim/mobject/geometry.py

```diff
@@ -2264,18 +2264,18 @@ class Angle(Arc, Elbow):
 
     Parameters
     ----------
-    line1 : :class:`Line`
+    line1 :
         The first line.
-    line2 : :class:`Line`
+    line2 :
         The second line.
-    radius : :class:`float`
+    radius :
         The radius of the :class:`Arc`.
     quadrant : Sequence[:class:`int`]
         A sequence of two :class:`int` numbers determining which of the 4 quadrants should be used.
         The first value indicates whether to anchor the arc on the first line closer to the end point (1)
         or start point (-1), and the second value functions similarly for the end (1) or start (-1) of the second line.
```

Comment:
```suggestion
        or start point (-1), and the second value functions similarly for the 
        end (1) or start (-1) of the second line.
```
Make a line break here as the sentence is too big

Reply:
thanks!